### PR TITLE
Update ecosystem

### PR DESCRIPTION
**Pull request by dabacon**
Addresses #4538 

#4538 mentions "Application-related benchmarks" but I don't know what this refers to. @cognigami?

* reworded services to be by type of quantum computer supported since it was getting to be redundant sentences.

* Did not include rigetti since my understanding is that it is not yet complete.

* Question: should Stim be under simulators or libraries?

* Finally: missing any integrations that should be added?

### CODE REVIEW

File: docs/ecosystem.md [hunk 1]
```diff
@@ -38,20 +39,20 @@ The following document provides an ecosystem overview of how the various tools c
...
-|Company|Description|
+|Company|Type of Quantum Computer Supported|
 |--- |--- |
-|[Google QCS](https://quantumai.google/cirq/tutorials/google/start)|A quantum computing service based on superconducting qubits|
-|[Alpine Quantum Technologies](https://quantumai.google/cirq/tutorials/aqt/start)|A quantum computing service based on ion traps|
-|[Pasqal](https://quantumai.google/cirq/tutorials/pasqal/start)|A quantum cloud service based on neutral atoms|
-
+|[Google QCS](https://quantumai.google/cirq/tutorials/google/start)|Superconducting qubits|
+|[Alpine Quantum Technologies](https://quantumai.google/cirq/tutorials/aqt/start)|Trapped ions|
+|[Pasqal](https://quantumai.google/cirq/tutorials/pasqal/start)|Neutral atoms|
```
Comment (viathor) [inline docs/ecosystem.md:46]: 404, but works after s/start/getting_started/

(Applies to AQT, too.)

File: docs/ecosystem.md [hunk 2]
```diff
@@ -38,20 +39,20 @@ The following document provides an ecosystem overview of how the various tools c
...
+|[Microsoft Azure Quantum](https://quantumai.google/cirq/tutorials/azure-quantum/getting_started_ionq)|Trapped ions (Honeywell and IonQ)|
 
 
 ## High performance quantum circuit simulators
 
 |Name|Main sponsor|Description|
 |--- |--- |--- |
 |[qsim](https://github.com/quantumlib/qsim)|Google|An open-source, high performance circuit simulator for Schrödinger simulations|
-|[qflex](https://github.com/ngnrsaa/qflex)|NASA|An open-source, high performance circuit simulator for Feynman-based simulations optimized for GPUs|
 |[quimb](https://github.com/jcmgray/quimb)|Johnny Gray|An open-source, high performance circuit simulator using tensor-networks|
```
Comment (viathor) [inline docs/ecosystem.md:56]: Out of curiosity: Why remove qflex?

**Review by viathor — APPROVED** [review]

File: docs/ecosystem.md (hunk 2 above)
Comment (MichaelBroughton) [reply docs/ecosystem.md:56]: cc: @95-martin-orion

Comment (95-martin-orion) [reply docs/ecosystem.md:56]: The motivation for this was to focus communications around qsim as the "standard" simulator for Cirq. Separately, while qFlex was originally Cirq-compatible, it hasn't been maintained - I'm doubtful that it works with modern Cirq.

**Comment by dabacon** [comment]
I've asked for feedback from PM and doc leads will wait for that response if there is more to update.

File: docs/ecosystem.md (hunk 2 above)
Comment (vcatherine) [reply docs/ecosystem.md:56]: +1 thanks Orion

**Comment by vcatherine** [comment]
Did not include rigetti since my understanding is that it is not yet complete.
-> suggest to add it, cirq-rigetti is working and Rigetti has confirmed they intend to maintain it

Question: should Stim be under simulators or libraries?
-> recommend adding STIM both under ecosystem and under simulators for now (you can create a circuit in Cirq for STIM). 

Finally: missing any integrations that should be added?
-> suggest to add NVIDIA cuQuantum here. There may be other additions at the end of the year, but if that happens we can create a new PR.

**Comment by vcatherine** [comment]
As mentioned in #4538, please add under Quantum Cloud Services: Microsoft Azure Quantum :)

**Comment by dabacon** [comment]
Thanks @vcatherine have added all of these.

**Pull request merged**